docs/plugins/virtual_environment_docs/5-survey.md

```diff
@@ -0,0 +1,115 @@
+# Survey/Q&A
+
+This guide contains questions that were submitted to our survey on testing.  
+
+## This guide covers:   
+* [Questions](#questions)  
+    - [Testing plugins](#testing-plugins)  
+    - [Testing widgets](#testing-widgets)  
+    - [Better signal or slot documentation or functionality](#better-signal-or-slot-documentation-or-functionality)  
+    - [Creating or deleting a point or shape layer](#creating-or-deleting-a-point-or-shape-layer)  
+    - [Github tests failing](#github-tests-failing)  
+    - [Cancelling a method](#cancelling-a-method)  
+    - [Testing environments](#testing-environments)  
+* [Topics](#topics)  
+    - [npe2](#npe2)  
+    - [Widget development](#widget-development)
+  
+## Questions
```

Comment:
I think we should get rid of `##Questions` and `##Topics` as headings. The distinction isn't really there.